src/sections/DonateSectionOne/DonateSectionOne.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import Button from 'components/Button/Button';
+import './DonateSectionOne.scss';
+import { ReactComponent as TreePlanting } from 'assets/tree-planting.svg';
+import { ReactComponent as CoinInHand } from 'assets/coin-in-hand.svg';
+import { ReactComponent as Car } from 'assets/car.svg';
+
+
+const DonateSectionOne = ({ data }) => {
+  const columnArray = Object.keys(data.body).map(i => data.body[i]);
+  const cols = columnArray.map((col, i) => {
+    const icons = [
+      <TreePlanting className='donate-section-one__icon'/>, 
+      <CoinInHand className='donate-section-one__icon'/>, 
+      <Car className='donate-section-one__icon'/>
+    ]
```

Comment:
Are these selectable in WP?

Reply:
They are now! I created selectable fields for the ACF